### ISSUE TITLE
upload_revision() takes two arguments

### ISSUE THREAD

**Issue by McKayMDavis** [CenterForOpenScience/osfr#58]
I've noticed that `upload_files()` calls `upload_revision()` and attempts to pass it three arguments. The function `upload_revision()` only takes two arguments.

**Comment by bgrich** [comment]
You are correct that `upload_revision()` only accepts 2 arguments. This is currently being fixed and should be addressed shortly.  

`upload_revision()` should only accept a file id. If passed a component id, there is some error handling code within `upload_revision()` that will tell the user to only use file ids for revisions. I agree that this makes it difficult to upload a revision without first having viewed it on OSF.

**Comment by tiernanmartin** [comment]
@bgrich It looks like you haven't had a chance to make this revision yet. When you do, could you also consider having `upload_revision()` return the file's url (as seen in `upload_new()`)?

I can submit a PR if for this if that would be helpful.

Thanks!
